packages/components/src/Autocomplete/Menu/Menu.tsx

```diff
@@ -21,6 +22,25 @@ export function Menu<
   inputRef,
   customRenderMenu,
 }: MenuProps<GenericOption, GenericOptionValue>) {
+  const { open } = useModalContext();
```

Comment:
somewhat of a nitpick, `open` does make sense as a concept within `Autocomplete` and `Autocomplete/Menu`

but this `open` isn't referring to either of those, so maybe we alias it as `parentModalOpen` or `modalOpen`?

pretty minor thing though, I don't feel that strongly about it